### PR TITLE
Add importants to bg helper classes

### PR DESCRIPTION
**Pull request by DaveVanVliet**
This PR adds `!important` to our bg helper classes so that it will override elements with high css specificity. 

This work began because bg helper classes were not properly overriding the spinner background while working on this bug: https://amaabca.visualstudio.com/DX_UX_Bugs/_workitems/edit/13992

This work also includes resized icons which were 1px off of either 96x96 or 192x192 to try and fix an issue where images are being slightly resized by chrome causing them to be blurry. They were run through kraken.io also to losslessly reduce the file size a bit. 

🎨 

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [x] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [ ] I've added new components to the style guide (or have a PR in to add them).
- [x] Any applicable version numbers have been updated.
- [ ] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [x] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [x] I have written a detailed PR message explaining what is being changed and why
- [x] I’ve linked to any relevant VSTS tickets
- [x] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

**Comment by ryanreddekopp** [comment]
📖

File: app/assets/stylesheets/layout/helper-classes.scss [hunk 1]
```diff
@@ -220,58 +220,58 @@
...
 }
 
 .orange-bg {
-  background-color: $orange;
+  background-color: $orange !important;
 }
 
 .red-bg {
-  background: $brand-red;
+  background: $brand-red !important;
```
Comment (ryanreddekopp) [inline app/assets/stylesheets/layout/helper-classes.scss]: I've noticed some inconsistencies with the attribute we are using to apply background color. Some classes use background, and some use background-color. Could we make these consistent?

File: app/assets/stylesheets/layout/helper-classes.scss [hunk 2]
```diff
@@ -220,58 +220,58 @@
...
-  color: $white;
+  color: $white !important;
 }
 
 .white-trans {
   /* stylelint-disable */
-  background-color: $white;
+  background-color: $white !important;
   -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=90)";
   background-color: rgba(255, 255, 255, 0.6);
```
Comment (ryanreddekopp) [inline app/assets/stylesheets/layout/helper-classes.scss]: Isn't the !important on the backup background-color going to overwrite the rgba background-color, which is the one we really want?

**Comment by ryanreddekopp** [comment]
💩

**Comment by ryanreddekopp** [comment]
Besides my other two comments, we should also think about changing the markup in the style guide after this passes. :)

File: app/assets/stylesheets/layout/helper-classes.scss (hunk 2 above)
Comment (DaveVanVliet) [reply app/assets/stylesheets/layout/helper-classes.scss]: Good point. I will remove it here completely actually since it it's a bg helper class

**Comment by DaveVanVliet** [comment]
👋

**Comment by ryanreddekopp** [comment]
🎨 🎨 🎨

**Pull request merged**